call-a-function-api-oci-clouddev.md

```diff
@@ -0,0 +1,772 @@
+2. Select your compartment.
+3. Click the **Copy** link for the OCID field. 
+
+Save the compartment OCID and name.
+
+### Collected Information
+
+Ensure you have the following information written down for the tutorial.
+
+* **Compartment Name: `<your-compartment-name>`**
+
+    Example: `my-compartment`
+
+* **Compartment ID: `<compartment-OCID>`**
+
+    Example: `ocid1.compartment.oc1.aaaaaaa...`
+
+* **VCN Name:** `<your-vcn-name>`
+
+    Example: `my-vcn`
+
+    Open the navigation menu and click **Networking**, and then click **Virtual Cloud Networks**. From the list of networks, select your VCN.
+
+* **VCN Public Subnet Name:** `<Public-Subnet-your-vcn-name>`
+
+    Example: `Public-Subnet-my-vcn`
+
+    Open the navigation menu and click **Networking**, and then click **Virtual Cloud Networks**. From the list of networks, select your VCN.
+
+## Perform Required Configuration
+
+Next you will configure what you need for the tutorial.
+
+### Create Functions Application
+
+To create application, follow these steps.
+
+1. Open the navigation menu and click **Developer Services**. Under **Functions**, click **Applications**.
+2. Select your compartment from the **Compartment** drop-down.
+3. Click **Create Application**.
+4. Fill in the form data.
+    * **Name:** `<your-app-name>`
+    * **VCN:** `<your-vcn-name>`
+    * **Subnets:** `<Public-Subnet-your-vcn-name>`
+5. Click Create.
+
+Your app is created.
+
+### Setup Ingress Rule for HTTPS
+
+1. Open the navigation menu and click **Networking**, and then click **Virtual Cloud Networks**.
+2. Click the name of the VCN you used to for your Oracle Functions application.
+3. With your new VCN displayed, click your **Public** subnet link. 
+
+    The public subnet information is displayed with the Security Lists at the bottom of the page.
+
+4. Click the Default Security List link or appropriate security list link. 
+
+    The default **Ingress Rules** for your VCN are displayed.
+
+5. Click Add Ingress Rules. 
+
+    An **Add Ingress Rules** dialog is displayed.
+
+6. Fill in the ingress rule with the following information. After all the data is entered, click Add Ingress Rules
+
+    Fill in the ingress rule as follows:
+
+    * **Stateless:** Checked
+    * **Source Type:** CIDR
+    * **Source CIDR:** 0.0.0.0/0
+    * **IP Protocol:** TCP
+    * **Source port range:** (leave-blank)
+    * **Destination Port Range:** 443
+    * **Description:** VCN for applications
+
+    After you click Add Ingress Rule, HTTPS connections are allowed to your **public subnet**.
+
+### Setup Policy for API Gateway Access to Functions
+
+Next, set up a policy which allows API Gateway to invoke functions.
+
+First, create a Dynamic Group for API Gateway.
+
+1. Open the navigation menu and click **Identity & Security**. Under **Identity**, click **Dynamic Groups**.
+2. Click **Create Dynamic Group**.
+3. Fill in the following information to define your dynamic group. 
```

Comment:
I'm not sure I understand these steps. The list items indicate a sequence with a subsequence? But the formatting breaks the list because the paragraph isn't nested inside of its parent, and the following list items aren't indented. Should this be nested? Should the paragraph be reworded as a list item? I'm confused, but I'd have to actually perform these steps to understand what we're going for here.